### PR TITLE
Post-it Component

### PR DESCRIPTION
**Pull request by gkojunior**
Created a Post-it component and css module file for styling. Created an image folder and added and image.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-  );
+	return (
+		<div className='App'>
+			<header className='App-header'>
+				<h1>Volunteer App</h1>
+				<PostIt />
+			</header>
+		</div>
+	);
 }
```
Comment (Leigh-K) [inline src/App.js]: Think the formatting bit stuffed here!

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-      <header className="App-header">
-        <h1>Volunteer App</h1>
-      </header>
-    </div>
-  );
+	return (
+		<div className='App'>
+			<header className='App-header'>
+				<h1>Volunteer App</h1>
+				<PostIt />
```
Comment (Leigh-K) [inline src/App.js]: Can you add a TODO: comment here to move this into the `Home` component once its merged in

File: src/components/PostIt.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+						<p>Need help asap!</p>
+					</div>
+                </div>
+
+			</div>
+		</div>
+	);
+};
+
+export default PostIt;
```
Comment (Leigh-K) [inline src/components/PostIt.js]: remove this

File: src/components/PostIt.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import styles from '../components/PostIt.module.css';
+import PostItImg from '../images/postItImg.jpg';
+
+const PostIt = () => {
```
Comment (Leigh-K) [inline src/components/PostIt.js]:
```suggestion
export const PostIt = () => {
```

**Review by Leigh-K — CHANGES_REQUESTED** [review]
You need to run prettier, let me know if you need a hand

**Comment by Leigh-K** [comment]
Also `PostIt` should be renamed to `Posting` I believe, folder then updated etc. 
And moved into a 
```
modules/home/components
```
As it will be shown on the homepage right?

**Comment by Leigh-K** [comment]
do you know why your package-lock is different from when in the repo?

**Comment by gkojunior** [comment]
Updates will be made asap! I needed to run "npm install" and I'm having funding issues!

**Pull request merged**